internal/command/test.go

```diff
@@ -997,6 +997,7 @@ func buildInputVariablesForTest(run *moduletest.Run, file *moduletest.File, conf
 				variables[name] = unparsedVariableValueExpression{
 					expr:       expr,
 					sourceType: tofu.ValueFromConfig,
+					ctx:        getEvalContextFromStates(states),
```

Comment:
Is there a reason to keep reconstructing this context inside the loop instead of constructing it once and passing the same context to all the variables here?